src/modules/dashboard/application/guards/__tests__/dashboard-rate-limit.guard.spec.ts

```
@@ -0,0 +1,235 @@
+    it('should handle case-insensitive widget path detection', async () => {
+      process.env.NODE_ENV = 'production';
+      mockRequest.path = '/dashboard/WIDGETS/stats';
+      
+      const result = await guard.canActivate(mockContext);
+      
+      expect(result).toBe(true);
+    });
+
+    it('should use default values when env vars are not set', async () => {
+      process.env.NODE_ENV = 'production';
+      delete process.env.RATE_LIMIT_DASHBOARD_WINDOW_MS;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)